### PR TITLE
Minchao

### PR DESCRIPTION
**Pull request by minchao-sun**
Add some dummy data.
Comments table structure changed.
add table `user` and change some references from `sys_users` to `user`

### CODE REVIEW

File: server/db/qmplus.sql [hunk 1]
```diff
@@ -616,6 +616,75 @@ CREATE ALGORITHM = UNDEFINED DEFINER = `root`@`localhost` SQL SECURITY DEFINER V
...
+  `updated_at` timestamp(0) NULL DEFAULT NULL,
+  `deleted_at` timestamp(0) NULL DEFAULT NULL,
+  `username` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NULL DEFAULT NULL,
+  `password` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NULL DEFAULT NULL,
+  `nick_name` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NULL DEFAULT 'QMPlusUser',
+  `header_img` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NULL DEFAULT 'http://www.henrongyi.top/avatar/lufu.jpg',
+  `phone_data` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NULL DEFAULT NULL,
+  `email` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NULL DEFAULT NULL,
+  PRIMARY KEY (`id`) USING BTREE,
+) ENGINE = InnoDB AUTO_INCREMENT = 12 CHARACTER SET = utf8 COLLATE = utf8_general_ci ROW_FORMAT = Compact;
```
Comment (lizihan021) [inline server/db/qmplus.sql:636]: AUTO_INCREMENT should be 1? If you set that to 12, the id will += 12 every time a new record is inserted.

File: server/db/qmplus.sql [hunk 2]
```diff
@@ -616,6 +616,75 @@ CREATE ALGORITHM = UNDEFINED DEFINER = `root`@`localhost` SQL SECURITY DEFINER V
...
+-- Records of users
+-- ----------------------------
+
+INSERT INTO `users` VALUES (
+	0001,
+	'2019-09-13 17:23:46', 
+	'2020-05-06 16:09:15', 
+	NULL, 
+	'Zihan', 
+	'ZihanLoveWen',
```
Comment (lizihan021) [inline server/db/qmplus.sql:648]: nice!

File: server/db/qmplus.sql [hunk 3]
```diff
@@ -616,6 +616,75 @@ CREATE ALGORITHM = UNDEFINED DEFINER = `root`@`localhost` SQL SECURITY DEFINER V
...
+
+INSERT INTO `users` VALUES (
+	0001,
+	'2019-09-13 17:23:46', 
+	'2020-05-06 16:09:15', 
+	NULL, 
+	'Zihan', 
+	'ZihanLoveWen', 
+	'Zihan Li', 
+	'https://github.com/zhaoolee/ChineseBQB/blob/master/048SpongeBob_海绵宝宝BQB/SpongeBob00001.jpg', NULL, NULL);
```
Comment (lizihan021) [inline server/db/qmplus.sql:650]: Those links might not work. An image link should provide only an image like this link http://qmplusimg.henrongyi.top/15887525450B978439-F04A-4a09-A8D3-DE7DE2677142.png . That link provides a web page instead. You can right-click the image in that web and get the link to the raw image like this one: https://github.com/zhaoolee/ChineseBQB/blob/master/048SpongeBob_%E6%B5%B7%E7%BB%B5%E5%AE%9D%E5%AE%9DBQB/SpongeBob00001.jpg?raw=true .

File: server/db/qmplus.sql [hunk 4]
```diff
@@ -660,9 +777,94 @@ CREATE TABLE `posts`  (
...
+	'怎么了',
+	1007,
+	NULL,
+	0005);
+INSERT INTO `posts` VALUES (
+	3009,
+	'2019-01-15 17:24:29',
+	'2019-02-19 19:27:29',
+	NULL,
+	'gbd',
```
Comment (lizihan021) [inline server/db/qmplus.sql:863]: gbd

File: server/db/qmplus.sql (hunk 2 above)
Comment (minchao-sun) [reply server/db/qmplus.sql:648]: nice

File: server/db/qmplus.sql (hunk 3 above)
Comment (minchao-sun) [reply server/db/qmplus.sql]: Done

File: server/db/qmplus.sql (hunk 4 above)
Comment (minchao-sun) [reply server/db/qmplus.sql:863]: god

Comment (minchao-sun) [reply server/db/qmplus.sql:863]: gbd

**Pull request merged**